tests/org.eclipse.n4js.hlc.tests/src/org/eclipse/n4js/hlc/tests/InstallCompileRunN4jscExternalImportsTest.java

```diff
@@ -47,6 +48,7 @@ public void deleteWorkspace() throws IOException {
 	 * running it with Common JS.
 	 */
 	@Test
+	@Ignore // remove @Ignore when IDE-2988 is done
```

Comment:
yo mean https://github.com/eclipse/n4js/pull/804 ? where is a GH issue for that PR?

Reply:
this is the one @qtran-n4 is working on. It will bring n4js-runtime-html5 into the shipped code